akka-cluster-typed/src/main/resources/reference.conf

```diff
@@ -27,6 +27,18 @@ akka.cluster.typed.receptionist {
   distributed-data.role = ""
 }
 
+akka.cluster.ddata.typed {
+  # The timeout to use for ask operations in ReplicatorMessageAdapter.
+  # This should be longer than the timeout given in Replicator.WriteConsistency and
+  # Replicator.ReadConsistency. The replicator will always send a reply within those
+  # timeouts so the unexpected ask timeout should not occur, but for cleanup in a
+  # failure situation it must still exist.
+  # If askUpdate, askGet or askDelete takes longer then this timeout a
+  # java.util.concurrent.TimeoutException will be thrown by the requesting actor and
+  # may be handled by supervision.
+  replicator-message-adapter-unexpected-ask-timeout = 20 s
```

Comment:
Nobody expects the ask timeout! 
<img width="163" alt="Screenshot 2019-06-14 at 10 30 45" src="https://user-images.githubusercontent.com/666915/59495399-7b343f80-8e8f-11e9-9998-850b5e7c7325.png">